core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -373,53 +375,111 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
     }
   }
 
-  // TODO: AS-627 re-enable, and potentially rewrite, in light of temp tables
-  it should "extra data in entity attribute temp table should not mess things up" in withEmptyTestDatabase {
-    val workspaceId: UUID = UUID.randomUUID()
-    val workspace = Workspace(
-      "test_namespace",
-      "test_name",
-      workspaceId.toString,
-      "bucketname",
-      Some("workflow-collection"),
-      currentTime(),
-      currentTime(),
-      "me",
-      Map.empty,
-      false)
+  // TODO: what are good values to test here for parallelism?
+  List(8, 16, 32, 64, 128, 256) foreach { parallelism =>
```

Comment:
could you use at least one odd number here, so that the tests saves a different number of entities to each of the two workspaces? I guess maybe that's not necessary if the test individually validates each saved entity ... what do you think?